docs/data/material/getting-started/templates/dashboard/components/CardAlert.tsx

```diff
@@ -16,7 +16,7 @@ export default function CardAlert() {
         <Typography color="text.secondary" variant="body2" sx={{ mb: 2 }}>
           Enjoy 10% off when renewing your plan today.
         </Typography>
-        <Button variant="contained" size="small" fullWidth>
+        <Button variant="outlined" size="small" fullWidth>
```

Comment:
I think the primary contained button would better reflect the priority here, mainly due to its contrast with the background and its role as the primary action in the card/drawer.